.gitpod.yml

```
@@ -3,7 +3,7 @@ github:
     pullRequestsFromForks: true
     addComment: true
 
-image: jsii/superchain:1-buster-slim-node16
+image: jsii/superchain:1-buster-slim-node18
```

According to https://github.com/aws/jsii/pull/4040 maybe we should use ecr public instead.


```suggestion
image: public.ecr.aws/jsii/superchain:1-buster-slim-node18
```